db/database.json

```diff
@@ -0,0 +1,20 @@
+{
+  "defaultEnv": {"ENV": "NODE_ENV"},
+  "local": {
+    "driver": "pg",
+    "user": "polkastats",
+    "password": "polkastats",
+    "host": "localhost",
+    "database": "polkastats",
+    "port": 5432
+  },
+  "production": {
+    "driver": "pg",
```

Comment:
Do we support other envs here?

Reply:
dev stage and prod envs are configured with NODE_END: production. Additional envs can be added here in case it's needed.

Reply:
If can configure all in 1 place then let's have only 1 configuration for all environments. We can add env vars in `.env` file

Reply:
We can put these variables into `.env.dafault` file.

Reply:
configuration updated. index.js files in api and backend modules passes NODE_ENV or 'local' value as env for db-migrate tool